plugins/core/IO/qPDALIO/src/LASFilter.cpp

```diff
@@ -140,8 +140,8 @@ LASFilter::LASFilter()
                     3.0f,	// priority
                     QStringList{ "las", "laz" },
                     "las",
-                    QStringList{ "LAS cloud (*.las *.laz)" },
-                    QStringList{ "LAS cloud (*.las *.laz)" },
+                    QStringList{ "LAS cloud with PDAL (*.las *.laz)" },
```

Comment:
That was to make things clearer on my side (temporarily). This can be removed of course.